src/BoilerPlate/Bullet.cpp

```diff
@@ -0,0 +1,52 @@
+#include "Bullet.hpp"
+
+#include <GL/glew.h>
+#include <SDL2/SDL_opengl.h>
+
+#include "Player.hpp"
+#include "MathUtilities.hpp"
+
+const int MAX_LIFE_TIME = 30;
+const float SHOT_FORCE = 10.0f;
+
+Bullet::Bullet(Player * shooter, int width, int height)
```

Comment:
It'd be great to make it shoot from the point of the shit rather than the center, not a requirement but a good exercise. ;-)